haas/cli.py

```diff
@@ -129,6 +134,7 @@ def serve(port):
     # We need to import api here so that the functions within it get registered
     # (via `rest_call`), though we don't use it directly:
     from haas import model, api, rest
+#    server.api_server_init() # Copied from b4cli.py
```

Comment:
Why comment this out?

If you intend to remove it permanently, just remove it. If folks want to see the way it was, they can use version control :)